Intern/rayx-core/src/Core.h

```diff
@@ -33,7 +33,6 @@
 #else
 #define RAYX_API
 #endif
-#pragma warning Unknown dynamic link import / export semantics.
```

Comment:
here we could add a check for clang instead

and in CMakeLists add the define for clang like so:
```
elseif(CMAKE_CXX_COMPILER_ID STREQUAL "Clang")
    set(COMPILE_PLATFORM RAYX_PLATFORM_CLANG)
```